s/sentry-python/build_info.json

```diff
@@ -1,16 +1,15 @@
 {
-  "maintainer": "shubham-garud",
+  "maintainer": "stutiibm",
   "package_name": "sentry-python",
   "github_url": "https://github.com/getsentry/sentry-python",
-  "package_dir": "s/sentry-python/",
+  "version": "2.7.1",
   "default_branch": "master",
-  "build_script": "sentry-python_ubi_8.4.sh",
-  "version": "1.12.1",
+  "package_dir": "s/sentry-python/",
+  "build_script": "sentry-python_ubi_9_3.sh",
   "docker_build": false,
-  "1.12.*, 1.11.*": {
-    "build_script": "sentry-python_1.12.1_ubi_8.7.sh"
-  },
-  "0.*.*": {
-    "build_script": "sentry-python_ubi_8.4.sh"
-  }
-}
\ No newline at end of file
+  "validate_build_script": true,
+  "use_non_root_user": true,
+  "2.*": {
```

Comment:
lets keep old build script supported for old version.. or new script works for 1.12.1 also?

Reply:
Added support for older script